Formula/digdag.rb

```diff
@@ -11,13 +12,13 @@ class Digdag < Formula
 
   bottle :unneeded
 
-  depends_on java: "1.8"
+  depends_on "openjdk@8"
 
   def install
     libexec.install "digdag-#{version}.jar"
     (libexec/"bin").write_jar_script libexec/"digdag-#{version}.jar", "digdag"
     (libexec/"bin/digdag").chmod 0755
-    (bin/"digdag").write_env_script libexec/"bin/digdag", Language::Java.java_home_env("1.8")
+    (bin/"digdag").write_env_script libexec/"bin/digdag", Language::Java.java_home_env("8")
```

Comment:
I believe this should be kept has 1.8. Does this work properly in `brew irb`?

Reply:
@jonchang 
Yes, `brew irb` worked as below

``` sh
$ brew irb
==> Interactive Homebrew Shell
Example commands available with: brew irb --examples

irb(main):001:0>
```

Reply:
I just tested it myself and this needs to be changed back.

```console
% brew irb
==> Interactive Homebrew Shell
irb(main):001:0> Language::Java.java_home_env("8")
=> {:JAVA_HOME=>"$(/usr/libexec/java_home --version 8 --failfast 2>/dev/null)"}
irb(main):002:0> Language::Java.java_home_env("1.8")
=> {:JAVA_HOME=>"/usr/local/opt/openjdk@8/libexec/openjdk.jdk/Contents/Home"}
```

Reply:
What about without specifying a version?

Reply:
```bash
$ brew irb
==> Interactive Homebrew Shell

irb(main):001:0>  Language::Java.java_home_env("1.8")
=> {:JAVA_HOME=>"/usr/local/opt/openjdk@8/libexec/openjdk.jdk/Contents/Home"}
irb(main):002:0>  Language::Java.java_home_env
=> {:JAVA_HOME=>"/usr/local/opt/openjdk/libexec/openjdk.jdk/Contents/Home"}
```